core/src/main/java/org/sonarsource/sonarlint/core/plugin/PluginClassLoaderDef.java

```diff
@@ -38,6 +39,8 @@
   private final String basePluginKey;
   private final Map<String, String> mainClassesByPluginKey = new HashMap<>();
   private final List<File> files = new ArrayList<>();
+  private final List<URL> urls = new ArrayList<>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "urls" private field. ([squid:S1068](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&pullRequest=165&issues=AWNE3mTC2jWE89Wx-sPf&open=AWNE3mTC2jWE89Wx-sPf)

[ISSUE_KEY]: AWNE3mTC2jWE89Wx-sPf